tensorflow_addons/text/crf.py

```diff
@@ -534,6 +535,10 @@ def crf_decode(
                   Contains the highest scoring tag indices.
       best_score: A [batch_size] vector, containing the score of `decode_tags`.
     """
+    if tf.__version__[:3] == "2.4":
+        warnings.warn(
+            "CRF Cecoding does may work in TF2.4. The bug has since been fixed in tensorflow/tensorflow##45534"
+        )
```

Comment:
I don't see anyway around this unfortunately.

Reply:
If I understand correctly it is an issue for `KerasTensor`. Is it possible to check this tensor type and only raise an error for this case?

Reply:
So KerasTensor is a private class and we'd have to  add crf.py to the exception list that we're not trying to grow:
https://github.com/tensorflow/tensorflow/issues/44613

I've updated the warning a bit, and since this will quickly throw an error if they are indeed KerasTensors I'm alright with it if everyone else is. Agree this isn't the best UX, but I also don't know why the fix didn't get cherry-picked into TF2.4

Reply:
The patch is not cherrypicked in TF2.4. I think the warning is `CRF Cecoding does may work in TF2.5. ...`?